mastering/build_files.py

```diff
@@ -1,5 +1,24 @@
 import os
-from ufoProcessor import build as upBuild
+import shutil
+from plistlib import dump as plDump
+from ufoProcessor import DesignSpaceProcessor
+from fontParts.fontshell import RFont
+from fontTools.designspaceLib import DesignSpaceDocument
+from defcon import Font
+
+# Family specific data
+
+weightMap = {
+             "Black": 900,
+             "UltraBold": 850,
+             "ExtraBold": 800,
+             "Bold": 700,
+             "SemiBold": 600,
+             "Medium": 500,
+             "Regular": 400,
+             "Light": 300,
+            }
```

Comment:
I think we should probably update these as the following, unless these need to stay below 900, for some reason:

```
weightMap = {
             "ExtraBlack": 1000,
             "Black": 900,
             "ExtraBold": 800,
             "Bold": 700,
             "SemiBold": 600,
             "Medium": 500,
             "Regular": 400,
             "Light": 300,
            }
```

Reply:
I usually don't set things at 1000; as that just came into the spec with version 1.8 (2016) but I guess it'll be fairly safe now. Will change.